sphinx_rtd_theme/layout.html

```diff
@@ -215,11 +215,7 @@
 
   <script type="text/javascript">
       jQuery(function () {
-          {% if theme_sticky_navigation|tobool %}
-          SphinxRtdTheme.Navigation.enableSticky();
-          {% else %}
-          SphinxRtdTheme.Navigation.enable();
-          {% endif %}
+          SphinxRtdTheme.Navigation.enable({{ theme_sticky_navigation|tobool }});
```

Comment:
This isn't quite right. I guess you could pass through `|lower` or something...

<img width="437" alt="screen shot 2018-04-30 at 1 38 31 pm" src="https://user-images.githubusercontent.com/185043/39449057-e3ee7bdc-4c7b-11e8-8f59-6a2d503dd4d3.png">

Reply:
Ah yes, good catch, updated and rebased with 68040a1.

I guess this worked because I didn't test a project overriding `sticky_navigation` in their conf.py. That is, i got the default `true` or `undefined`.